core/json/src/test/scala/net/liftweb/json/JsonPrintingSpec.scala

```diff
@@ -29,14 +29,12 @@ import org.scalacheck.Prop.forAll
 object JsonPrintingSpec extends Specification  with JValueGen with ScalaCheck {
   "JSON Printing Specification".title
 
-  import scala.text.Document
-
   "rendering does not change semantics" in {
-    val rendering = (json: Document) => parse(Printer.pretty(json)) == parse(Printer.compact(json))
+    val rendering = (json: JValue) => parse(JsonAST.prettyRender(json)) == parse(JsonAST.compactRender(json))
     check(forAll(rendering))
   }
 
   private def parse(json: String) = scala.util.parsing.json.JSON.parseRaw(json)
 
-  implicit def arbDoc: Arbitrary[Document] = Arbitrary(genJValue.map(render(_)))
+  implicit def arbDoc: Arbitrary[JValue] = Arbitrary(genJValue)
```

Comment:
I think so, but admittedly the changes I made here required so much refactoring I didn't scrutinize them too much. Refactoring these tests would be A Good Thing™ - but given the breadth of this change I'd like to avoid doing so now. It somewhat invalidates the usefulness of the tests toward checking the branch if we go changing them in any meaningful way in the branch itself.